### ISSUE TITLE
Need to Quick Calibrate at the beginning of every session. 

### ISSUE THREAD

**Issue by enqui** [ValveSoftware/steamvr_unity_plugin#352]
We need a way to automatically "Quick Calibrate" at the beginning of every session. 

The camera often moves higher than intended, leading to, either the player floating, or their head coming out of top of the vehicles, or caves etc. Using "Quick Calibrate" from the settings fixes that, but we can't have the user doing this or having someone doing this for the user every time a new session starts with that issue.

**Comment by RichBBVR** [comment]
We have the same issue with our experience. Is it still happening with you? Ours happens probably 1 out of every 4 or 5 times, then a restart of SteamVR calibrates everything back to normal. Sometimes we get room rotation or tilt issues too, may be separate cause. Our users have to wait for a restart which really interrupts the immersion, so not ideal :)

**Comment by enqui** [comment]
yes. We have a seated experience and we do not want to leave this to luck.  We want to be able to calibrate each and every time the experience starts for the audience and the camera to start exactly where we need it each and every time. There is no opportunity for randomness.

**Comment by RichBBVR** [comment]
Curious, have you tried this tool https://github.com/alsoknownasfoo/ViveArcadeTool/releases

It's supposed to "lock in" the universe, which should include height but may not. We have been unsuccessful in stabilizing the tool for enterprise system but may be worth testing. We are testing many variables to help lock the universe using different methods - I'll let you know if we discover anything concrete.

**Comment by h3teroerectus** [comment]
@RichBBVR That's more or less what our adaptation of Floor Fix does, but again if the SteamVR tracking universe if off, or there is drift in the trackers, it's impossible to know for sure.

**Comment by RichBBVR** [comment]
@enqui which devices, versions, and how many base stations are you using? We have the Vive Pro, 2 Controllers (2.0 Trackers used as controllers), 1 x 2.0 Tracker for positional mapping, and 3 x 2.0 Base Stations.

**Comment by enqui** [comment]
Using OpenVR and WMR devices.

**Comment by zite** [comment]
This sounds more like an OpenVR issue than a plugin issue. @kisak-valve can you move this issue?
